draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1270,20 +1274,51 @@ The "rtp-mux-quic" string identifies RoQ:
   Specification:
   : This document
 
-## RoQ Error Codes
+## RoQ Error Codes Registry {#iana-error-codes}
+
+This document establishes a registry for RoQ error codes. The "RoQ Error Codes"
+registry manages a 62-bit space.
+
+The new error codes registry created in this document operates under the QUIC
+registration policy documented in {{Section 22.1 of !RFC9000}}. This registry
+includes the common set of fields listed in {{Section 22.1.1 of !RFC9000}}.
+
+Permanent registrations in this registry are assigned using the Specification
+Required policy ({{!RFC8126}}), except for values between 0x00 and 0x3f (in
+hexadecimal; inclusive), which are assigned using Standards Action or IESG
+Approval as defined in {{Sections 4.9 and 4.10 of !RFC8126}}.
+
+Registrations for error codes are required to include a description of the error
+code. An expert reviewer is advised to examine new registrations for possible
+duplication with existing error codes.
+
+In addition to common fields as described in Section {{Section 22.1 of
+!RFC9000}}, this registry includes two additional fields. Permanent
+registrations in this registry MUST include the following fields:
+
+Name:
+: A name for the error code.
```

Comment:
> A short mnemonic for the parameter

Reply:
Also taken from HTTP/3. I find *A name for the error code* much easier to understand, especially since the codes are not called parameters anywhere else.